### ISSUE TITLE
Failed start wechat/weixin

### ISSUE THREAD

**Issue by ZedG2** [sandboxie-plus/Sandboxie#3354]
### Describe what you noticed and did

1. Start wechat.exe inside sandbox
2. wechat.exe and mmcrashpad_handler64.exe processes find.
3. nothing to shown, high cpu usage before kill it.

### How often did you encounter it so far?

Everytime

### Affected program

WeChat PC 3.9.7

### Download link

https://dldir1.qq.com/weixin/Windows/WeChatSetup.exe

### Where is the program located?

The program is installed only inside a sandbox (NOT in the real system anyway).

### Expected behavior

Wechat started.

### What is your Windows edition and version?

Windows 11 Pro 22h2

### In which Windows account you have this problem?

A Microsoft account (Administrator).

### Please mention any installed security software

Defender

### What version of Sandboxie are you running?

1.11.3

### Is it a new installation of Sandboxie?

I just updated Sandboxie from a previous version (I don't remember which one).

### In which sandbox type you have this problem?

In a standard isolation sandbox (yellow sandbox icon).

### Can you reproduce this problem on a new empty sandbox?

I can confirm it also on a new empty sandbox.

**Comment by offhub** [comment]
Can you check if it's fixed in the latest version?

https://github.com/sandboxie-plus/Sandboxie/releases/latest

**Comment by ZedG2** [comment]
> Can you check if it's fixed in the latest version?
> 
> https://github.com/sandboxie-plus/Sandboxie/releases/latest

same in the latest version.

![image](https://github.com/sandboxie-plus/Sandboxie/assets/7886856/9671a5bd-3e77-4ad4-a6e3-d81f1e480c3c)

**Comment by ttimasdf** [comment]
Did you update WeChat recently? I'm running WeChat 3.9.6.47 with sandboxie 1.11.4 without issues

**Comment by ZedG2** [comment]
> Did you update WeChat recently? I'm running WeChat 3.9.6.47 with sandboxie 1.11.4 without issues. 

Yes, older version is work. But the "TencentOCR.exe" has high cpu usage.

**Comment by Thundercup** [comment]
I have same problem on wechat and dingtalk (钉钉), they just crash  when start without UAC, even emulate admin rights does not work.

**Comment by asky88** [comment]
Encountered the same problem. A strange thing is that WeChat can start normally in a sandbox with data protection, but not in a sandbox without data protection.

**Comment by cxw620** [comment]
Encountered the same problem. Any idea on this issue?
